### PR TITLE
Stabalize

### PR DESCRIPTION
**Pull request by zachChilders**
This PR adds a test suite that will automatically deploy, test, and cleanup our infrastructure to ensure that nothing is broken.

- [x] Test Suite
- [x] Baseline Test Application
- [x] Parametric Deployment Script
- [x] Runs clean

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,5 +16,8 @@ jobs:
         shell: pwsh
         run: ./.github/scripts/Assert-Linting
       - name: Test code
+        env:
```
Comment (zachChilders) [inline .github/workflows/ci.yml:19]: These are plumbed in from the secrets found in this repo's settings.

File: deploy.ps1 [hunk 2]
```diff
@@ -159,43 +166,66 @@ $k8sReqs = @(
...
         Test     = { [boolean] (kubectl describe nodes | grep kured) }
         Set      = {
             kubectl apply -f https://github.com/weaveworks/kured/releases/download/1.2.0/kured-1.2.0-dockerhub.yaml
         }
     },
     @{
-        Name     = "Harden Cluster"
-        Describe = "Apply security policy"
-        Test     = { kubectl get psp } # Improve tests
+        Describe = "Create DNS Name"
```
Comment (zachChilders) [inline deploy.ps1:189]: We have DNS names now, for testing and HTTPS purposes

File: tf/enclave/clean.ps1 [hunk 3]
```diff
@@ -0,0 +1,31 @@
+param(
```
Comment (zachChilders) [inline tf/enclave/clean.ps1:1]: This is a cleanup script that resets both your local system and azure

**Comment by zachChilders** [comment]
Enables #41

**Review by lisnter — APPROVED** [review]

**Review by lisnter — APPROVED** [review]
Good comments.  :-)  And abstracting things.

**Review by lisnter — APPROVED** [review]

File: deploy.ps1 [hunk 4]
```diff
@@ -223,7 +223,20 @@ $k8sReqs = @(
     #         az extension update --name aks-preview
 
     #         # Apply default policy
-    #         az aks update --resource-group sbd --name sbd --enable-pod-security-policy
+    #         az aks update --resource-group sbd --name sbd --enable-pod-security-polic
```
Comment (lisnter) [inline deploy.ps1:226]: Why change from -policy to -polic?

**Review by lisnter — CHANGES_REQUESTED** [review]
See comment in-line.

**Review by lisnter — APPROVED** [review]

File: deploy.ps1 (hunk 4 above)
Comment (zachChilders) [reply deploy.ps1:226]: typo, this is commented out and is the next order of business to get integrated now.

**Pull request merged**